sql/parser2/eval.go

```diff
@@ -639,3 +687,23 @@ func evalCaseExpr(expr *CaseExpr, env Env) (Datum, error) {
 	}
 	return null, nil
 }
+
+func evalTupleEQ(ldatum, rdatum Datum) (Datum, error) {
+	left := ldatum.(dtuple)
+	right := rdatum.(dtuple)
+	if len(left) != len(right) {
+		return dbool(false), nil
+	}
+	for i, lval := range left {
```

Comment:
nit: i'd use a normal for-loop here, but im not sure what the broadly accepted idiom here is